shmem.c

```diff
@@ -105,18 +105,18 @@ void newOverTimeClient() {
 }
 
 void addOverTimeClientSlot() {
-	// For each client slot, add an overTime slot
-	for(int i = 0; i < overTimeClientCount; i++) {
-		// Get the name of the shared memory
-		char *name = clientShmName(i);
```

Comment:
There seems to be no need for the `name` buffer here, so I removed this code. Please verify this @Mcat12

Reply:
Correct